README.md

```diff
@@ -4,18 +4,61 @@ keywords: machine, hetzner, Hetzner Cloud, driver
 title: Hetzner Cloud
 ---
 
-Create Docker machines on [Hetzner Cloud](https://docs.hetzner.cloud/).
+Create [Docker machines](https://github.com/docker/machine) on [Hetzner Cloud](https://docs.hetzner.cloud/).
 
 You need to create a project-sepcific access token under `Access` > `API Tokens` in the project control panel
 and pass that to `docker-machine create` with the `--hetzner-api-token` option.
 
+## Building
```

Comment:
Could you move the whole section to the end please? The README was originally copied from some other driver (DOcean I think) and adapted to match the format of 'standard' driver documentations. Thus I'd like to keep specific stuff at the end.

Reply:
Yes, I could move it to the end. On the other hand this driver is not included in docker-machine and right now you need to build it yourself before you can use it. So the proposed document structure reflects the order of work that users need to do to be able to use the driver. I.e. I think it makes sense to have build instructions at the top for now.

When a binary of the driver becomes available, build instructions might move to the bottom and instructions how to obtain / use the binary might take its place.

Does that sound reasonable?